docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1652,7 +1652,7 @@ Ingesters keep, on their persistent disk, the blocks compacted from TSDB head un
 
 The blocks retained in the ingesters can be used in case the compactor generates corrupted blocks and the source blocks, shipped from ingesters, have already been hard deleted from the bucket.
 
-How to manually blocks from ingesters to the bucket:
+How to manually ship blocks from ingesters to the bucket:
```

Comment:
I'm not 100% sure about my use of the word "ship" here - if there's a better verb to use, happy to change it.

Reply:
I would probably use "upload". In ingester code we dp refer to this upload as "shipping", and it's done by "shipper", but in the documentation I don't think we use that term.

Reply:
Yes, please change to "upload".

Reply:
"upload" makes sense to me. I'll also update the line above to use "uploaded" instead of "shipped" to keep things consistent.